imports/plugins/core/orders/client/templates/workflow/cancelOrder.js

```diff
@@ -0,0 +1,108 @@
+import { Meteor } from "meteor/meteor";
+import { Template } from "meteor/templating";
+
+const validateComment = (comment) => {
```

Comment:
Yes its possible. There could also be edge cases where a little extra details about the cancellation is needed.